ui/src/app/base/components/Definition/Definition.tsx

```diff
@@ -27,8 +27,10 @@ const Definition = ({ label, children, description }: Props): JSX.Element => {
       </p>
       {description ? (
         <p aria-labelledby={id}>{description}</p>
-      ) : React.Children.toArray(children).length > 0 ? (
-        React.Children.toArray(children).map(
+      ) : React.Children.toArray(children).filter((child) => child !== "")
```

Comment:
Should this also filter other invalid nullish values (but not the valid ones like `0`)?

Reply:
React.Children.toArray handles that already (and there was already a test for null + undefined children in this file).

Reply:
https://github.com/canonical-web-and-design/maas-ui/blob/63f6eafdbc96ec3475b4d6d9ee14aabc37fcac3b/ui/src/app/base/components/Definition/Definition.test.tsx#L31-L33

Reply:
Ah great, thanks! Sorry, I missed that.